src/main/java/app/services/MovieService.java

```diff
@@ -54,18 +54,19 @@ public Set<CastMemberDTO> getCastMembersByMovieId(int id) {
                 } else {
                     return castMembersSet;
                 }
+
             } else {
                 System.out.println("GET request failed. Status code: " + response.statusCode());
             }
         } catch (IOException | InterruptedException e) {
             e.printStackTrace();
         }
+
         return null;
     }
 
     public Set<MovieDTO> getMoviesByCountry(String country) {
         try {
-            HttpClient httpClient = HttpClient.newHttpClient();
```

Comment:
The HttpClient declaration can be beneficial to readability and is probably not a big deal to refactor/remove. So in the future it might be better to leave it as it doesn't have a big impact but can help other team members with flow and readbility

Reply:
Was changed to have the same structure as the other methods in the class for less inconsistency and to somewhat follow same code standard.

Reply:
A code standard for inlined variables was not really discussed. It does not really give a worse result so inlining should probably not be taken too serious at this point as other team members still rely on declaring variables to better understand the code. So unless it has been discussed, necessary or arguably better, it should probably be avoided to mess too much with it.

Reply:
No true. No code standard was actually ever discussed only briefly mentioned it would be good to follow same structure.

Reply:
I do agree that following the same structure would be best but it might still be too early in the education to enforce it as a standard. But since this is highly subjective I won't state that it's wrong. I do agree that the same structure should be a standard but I still believe hat other team members should be taken into consideration when condensing and inlining code. It's not important so I won't be pursuing it any further